spec/services/exports/export_distributions_csv_service_spec.rb

```diff
@@ -108,16 +99,19 @@
     end
 
     context 'when a new item is added' do
-      let!(:original_columns_count) { organization.items.size + non_item_headers.size }
       let(:new_item_name) { "new item" }
       before do
-        create(:item, name: new_item_name, organization:)
+        # let(:distributions) creates 5 items.
+        # make sure they are all created before we record the original column count
+        distributions
+        @original_columns_count = organization.items.size + non_item_headers.size
```

Comment:
In this case we _know_ what the number should be for that count. It should always be 15. Just hardcode it.